docs/getting-started.md

```diff
@@ -91,38 +91,39 @@ Explore our CLI to find more queries you can run.
 ### Submitting a transaction
 
 
-1. Generate an account
+1. Generate a key-pair
```

Comment:
I think this section would benefit from more clearly explaining the relation ship between accounts and key pairs stored locally. Specifically we should make it clear that an account lives on chain and is controlled by a key pair. An account has an address which is determined by the key pair. I’d suggest saying “account address for a key pair” instead of “key pair address” to make this more obvious.

Reply:
> I think this section would benefit from more clearly explaining the relation ship between accounts and key pairs stored locally

Yes, absolutely.

It actually would make more sense to have it in the next sub-section, `Request RADs`, where this understanding becomes important, don't you think? Will commit a suggestion, can you have a look?